release-team/role-handbooks/docs/Release-Timeline.md

```diff
@@ -695,12 +718,35 @@ ef6c80a0e - Kaitlyn Barnard, 5 hours ago : 1.14 Blog Post (#13400)
 851ef58fa - Jim Angel, 5 hours ago : Official 1.14 Release Docs (#13174)
 28dd4d515 - evilyeti, 7 hours ago : Fixed broken link in E2E Testing blog
 ```
-If that looks how you expect:
+If that looks how you expect, the following example is from the 1.14 release:
 ```shell
 git tag -a snapshot-final-v1.13 28dd4d515 -m "Release 1.13 final snapshot"
 git tag -a snapshot-initial-v1.14 851ef58fa -m "Release 1.14 initial snapshot"
 git push --tags origin master
 ```
+If the `git log` command does not look how you expect then you can use the following to confirm the commit hash before the [integration branch]:
```

Comment:
Can we just get rid of L713-726, and use the most recent example? The example log causes more confusion and I think it would be better to remove it for clarity.